Userland/Utilities/file.cpp

```diff
@@ -51,6 +52,39 @@ static Optional<DeprecatedString> image_details(StringView description, StringVi
     return builder.to_deprecated_string();
 }
 
+static Optional<DeprecatedString> audio_details(StringView description, StringView path)
+{
+// If serenity_main is defined as a macro, we're compiling for BuggieBox.
```

Comment:
Why are we handling these cases differently?

Reply:
There is no statically linked version of LibAudio so BuggieBox can't access it. I did not want to chase that yak and I believe there is little utility to knowing the sample rate of a random audio file while you're fixing your Serenity installation.

Reply:
Wait, is this actually even needed? I apply the below diff on top of your branch, and BuggieBox compiles and runs file - I don't see anything that indicates we need static libraries here.
```diff
diff --git a/Userland/BuggieBox/CMakeLists.txt b/Userland/BuggieBox/CMakeLists.txt
index 200daa5142..fcd294ed8b 100644
--- a/Userland/BuggieBox/CMakeLists.txt
+++ b/Userland/BuggieBox/CMakeLists.txt
@@ -41,7 +41,7 @@ set(utility_srcs
 
 serenity_bin(BuggieBox)
 target_sources(BuggieBox PRIVATE main.cpp)
-target_link_libraries(BuggieBox PRIVATE LibMain LibShell LibCompress LibCore LibCrypto LibGfx LibLine LibRegex)
+target_link_libraries(BuggieBox PRIVATE LibMain LibShell LibCompress LibCore LibCrypto LibGfx LibLine LibRegex LibAudio)
 
 foreach(file IN LISTS utility_srcs)
    buggiebox_utility(${file})
diff --git a/Userland/Utilities/file.cpp b/Userland/Utilities/file.cpp
index 9a5bcc1b7f..9856e56f49 100644
--- a/Userland/Utilities/file.cpp
+++ b/Userland/Utilities/file.cpp
@@ -54,10 +54,6 @@ static Optional<DeprecatedString> image_details(StringView description, StringVi
 
 static Optional<DeprecatedString> audio_details(StringView description, StringView path)
 {
-// If serenity_main is defined as a macro, we're compiling for BuggieBox.
-#if defined(serenity_main)
-    return description_only(description, path);
-#else
     auto loader_or_error = Audio::Loader::create(path);
     if (loader_or_error.is_error())
         return {};
@@ -82,7 +78,6 @@ static Optional<DeprecatedString> audio_details(StringView description, StringVi
         builder.appendff(" by {}", all_artists.release_value());
 
     return builder.to_deprecated_string();
-#endif
 }
```

Reply:
This does mean that BuggieBox now has another dependency that is growing by the day, but if you have no problem with it I will of course do the "easy" thing :^)

> Also note `audio_details` appends an extra newline compared to other details:

That's an unintentional display bug since the second line should contain metadata, but is empty here. Will be fixed of course.